apps/docs/scripts/components-with-animation.js

```diff
@@ -11,7 +11,6 @@ export const supportedComponents = [
   'Pillar',
   'SectionIntro',
   'Stack',
-  'Pillar',
```

Comment:
Was this an intentional deletion?

Reply:
Yeah it's a dupe of the one above (line 11).. no idea how this got inserted here, maybe post-merge conflict 🤷‍♂️.  This file itself is generated, and isn't being built deterministically right now.. so you can ignore changes to this file for time being until that's fixed.